low-level/files/opt/seagate/sspl/conf/manifest.yaml

```diff
@@ -14,4 +14,4 @@
 # cortx-questions@seagate.com.
 
 support_bundle:
-    - /opt/seagate/cortx/sspl/bin/manifest_support_bundle
\ No newline at end of file
+    - /usr/bin/manifest_support_bundle
```

Comment:
why path changed here, it needs to remain same

Reply:
For binary code we have created this symbolic link in /opt/seagate/cortx/sspl/bin/ but now we have created a symbolic link in /usr/bin/manifest_support_bundle for easier access to this script.